x-pack/plugins/uptime/server/rest_api/pings/journey_screenshots.ts

```diff
@@ -39,22 +35,13 @@ export const createJourneyScreenshotRoute: UMRestApiRouteFactory = (libs: UMServ
   handler: async ({ uptimeEsClient, request, response }) => {
     const { checkGroup, stepIndex } = request.params;
 
-    let result: ScreenshotReturnTypesUnion | null = null;
-    try {
-      result = await libs.requests.getJourneyScreenshot({
-        uptimeEsClient,
-        checkGroup,
-        stepIndex,
-      });
-    } catch (e) {
-      return response.customError({ body: { message: e }, statusCode: 500 });
-    }
-
-    if (isFullScreenshot(result)) {
-      if (!result.synthetics.blob) {
-        return response.notFound();
-      }
+    const result: ScreenshotReturnTypesUnion | null = await libs.requests.getJourneyScreenshot({
+      uptimeEsClient,
+      checkGroup,
+      stepIndex,
+    });
 
+    if (isFullScreenshot(result) && !!result.synthetics?.blob) {
```

Comment:
Should we be using !! instead of Boolean? Just wondering. As I've been thinking about readability more often, I realize that it takes my brain a half a second more to process what !! means versus Boolean, but that may just be me.

Reply:
++ I agree, I'll switch it. I actually hate `!!`, so idk why I'm using it so much these days. It looks hacky and `Boolean()` is much nicer.

Reply:
So, I didn't notice this until I went to implement, but `Boolean()` doesn't work in this situation because it doesn't assure the TS compiler that `result.synthetics?.blob` is defined. I've opted instead for:

```typescript
typeof result.synthetics?.blob !== 'undefined'
```

Reply:
https://github.com/elastic/kibana/pull/103979/commits/ee6ea7979ffabc79429f2693dc21e9b5dc5e09f6